index.php

```diff
@@ -1,4 +1,28 @@
-
+<?php
+$page=$_GET['q'];
+// var_dump($page);
+if ($page == 2) {
+  header('location: http://sourav_domain.com/login_user.php');
+}
+elseif ($page == 3) {
+  header('location: http://sourav_domain.com/name.php');
+}
+elseif ($page == 4) {
+  header('location: http://sourav_domain.com/photo_upload.php');
+}
+elseif ($page == 5) {
+  header('location: http://sourav_domain.com/mobile.php');
+}
+elseif ($page == 6) {
+  header('location: http://sourav_domain.com/mail.php');
+}
+elseif ($page == 7) {
+  header('location: http://sourav_domain.com/marksheet.php');
+}
+elseif ($page == 8) {
+  header('location: http://sourav_domain.com/pdf_Marksheet.php');
+}
+?>
```

Comment:
Why we are redirecting user to the pages. Can't we just include the file?

Reply:
> Why we are redirecting user to the pages. Can't we just include the file?
now i do it by using include file method